pages/_language/events/_event/_chapter.vue

```diff
@@ -71,6 +71,17 @@
           >
             <story-map :mapbox-style="page.mapboxStyle" />
           </div>
+
+          <div
+            v-if="page.creditsTitle"
```

Comment:
also check if there's a `.creditsBody` and `.creditsLogos`?

Reply:
yes 👍